### PR TITLE
API1800 support for LE and interconnects

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
API1800 support for LE and interconnects

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: spec/unit/resource/api1800/synergy/logical_enclosure_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
```
Comment (nabhajit-ray) [inline spec/unit/resource/api1800/synergy/logical_enclosure_spec.rb:1]: add copyright

File: spec/unit/resource/api1800/c7000/logical_enclosure_spec.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
```
Comment (nabhajit-ray) [inline spec/unit/resource/api1800/c7000/logical_enclosure_spec.rb:1]: add copyright

File: spec/unit/resource/api1800/synergy/logical_enclosure_spec.rb (hunk 1 above)
Comment (yuvirani) [reply spec/unit/resource/api1800/synergy/logical_enclosure_spec.rb:12]: Done

File: spec/unit/resource/api1800/c7000/logical_enclosure_spec.rb (hunk 2 above)
Comment (yuvirani) [reply spec/unit/resource/api1800/c7000/logical_enclosure_spec.rb:12]: Done

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,8 +10,8 @@ This release extends support to OneView Rest API version 1800 (OneView v5.30) fo
 - Ethernet Network
 - FC Network
 - FCOE Network
-- Hypervisor Cluster Profile
-- Hypervisor Manager
```
Comment (VenkateshRavula) [inline CHANGELOG.md:14]: why to delete hypervisor resources..?

Comment (yuvirani) [reply CHANGELOG.md:14]: reverted back

**Pull request merged**